src/appendices/array-method-examples/slice-examples.rst

```diff
@@ -0,0 +1,35 @@
+.. _slice-examples:
+
+**slice** Examples
+===================
+
+The general syntax for this method is:
+
+::
+
+   arrayName.slice(starting index, ending index)
+
+This method copies selected entries of an array into a new array. ``slice``
```

Comment:
How about "This method copies a range of elements in an array into a new array"

Reply:
Also possibly make it more clear that both ways to use slice returns a new array.